src/cdk/listbox/listbox.ts

```diff
@@ -783,6 +796,11 @@ export class CdkListbox<T = unknown> implements AfterContentInit, OnDestroy, Con
    * @param event The focusout event
    */
   protected _handleFocusOut(event: FocusEvent) {
+    // Some browsers (e.g. Chrome and Firefox) trigger the focusout event when the user returns back to the document.
+    // To prevent losing the active item in this case, we store it in `_previousActiveItem` and restore it on the window `blur` event
+    // This ensures that the `activeItem` matches the actual focused element when the user returns to the document.
+    this._previousActiveItem = this.listKeyManager.activeItem;
```

Comment:
Instead of resetting this on each `focusout` event, would it make more sense to make something like this?

```
if (this. _activeItemOnWindowBlur) {
  this._activeItemOnWindowBlur = null;
  return;
}
```

Reply:
yeah you're right...I'll reset it.
what do you think we should name the class member? :))

Reply:
Done